docs/sources/plugins/developing/dataframe.md

```diff
@@ -98,11 +98,11 @@ Dimensions: 3 Fields by 2 Rows
 +---------------------+-----------------+-----------------+
 ```
 
-#### Time Series (Long Format)
+#### Time series in long format
 
 (Note: Currently supported on backend only: [Grafana Issue #22219](https://github.com/grafana/grafana/issues/22219)).
```

Comment:
Is this still valid?

Reply:
fixed

Reply:
Yea. I have not idea how we should deal with statements like this. It's bound to get out of sync. Perhaps we should just remove it?